packages/attendance/src/components/AttendanceComponent.js

```diff
@@ -557,49 +561,53 @@ export default function AttendanceComponent({
     setLoading({
       [dataObject.date + dataObject.id]: true,
     });
-    if (dataObject.attendanceId) {
-      attendanceRegistryService
-        .update(
-          {
-            id: dataObject.attendanceId,
-            attendance: dataObject.attendance,
-          },
-          {
-            onlyParameter: ["attendance", "id", "date", "classId"],
-          }
-        )
-        .then((e) => {
-          const newData = attendance.filter(
-            (e) =>
-              !(
-                e.date === dataObject.date &&
-                e.studentId === dataObject.studentId
-              )
-          );
+    if (moment().format("HH:MM") <= manifest?.["class_attendance.submit_by"]) {
```

Comment:
please review else part.
what will happen if current time has passed the submit by value? I think in current code user is not promoted with any message.